Dynatrace/dynatrace-operator
# Fix span titles for OTel.

## Description

The automatically attached span title was wrong. It always used `dtotel.StartSpan` instead of the actually calling function.

## How can this be tested?

Deploy the operator, deploy a Dynakube and check for distributed traces in your tenant.

## Checklist
